compiler/src/dmd/mars.d

```diff
@@ -631,6 +631,15 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             continue;
         }
 
+        if (arg == "-verrors=context")
+        {
+            global.params.v.printErrorContext = true; // Ensure this is enabled
+        }
+        else if (arg == "-verrors=none")
+        {
+            global.params.v.printErrorContext = false; // Ensure this is disabled
+        }
```

Comment:
this change should be made https://github.com/dlang/dmd/blob/master/compiler/src/dmd/mars.d#L1010 and not here

Reply:
done